src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1429 @@
+   {
+      UT_LOG_ERROR("Result: Failed to dsHdmiInScaleVideo:[%s]",ut_control_GetMapString(dsError_mapTable, ret));
+   }
+     
+   UT_LOG_INFO("Result : Passed dsHdmiInScaleVideo paramsx : %d, y : %d, heitht : %d , width : %d",
+                  x, y , width, height);
+
+   UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to select zoom mode.
+*
+* This test function is select zoom mode of selected HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 006@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_zoom_mode(void)
+{
+    gTestID = 6;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    int select;
+    dsVideoZoom_t mode;
+
+    printf("\n===================Select Zoom Mode================\n");
+    printf("\n**************Availabe Zoom Modes are****************\n");
+    for(dsVideoZoom_t i = dsVIDEO_ZOOM_NONE ; i < dsVIDEO_ZOOM_MAX ; i++)
+    {
+           UT_LOG_INFO("%s : %d", ut_control_GetMapString(dsVideoZoom_mapTable, i) , i);
+    }
+    printf("\n******************************************************\n");
+
+    readInput(&select);
+    if(select < 0 || select > dsVIDEO_ZOOM_MAX)
+    {
+      printf("\n invalid zoom mode selected \n");
+    }  
+    
+    mode = (dsVideoZoom_t)select;
+    ret = dsHdmiInSelectZoomMode(mode);
+    if(ret != dsERR_NONE)
+    {
+       UT_LOG_ERROR("Result: Failed to dsHdmiInSelectZoomMode Error:[%s]",ut_control_GetMapString(dsError_mapTable, ret));
+    }
+
+    UT_LOG_INFO("Result : dsHdmiInSelectZoomMode: %s, selected: %d",
+                 ut_control_GetMapString(dsVideoZoom_mapTable, mode), mode);
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+    
+}
+
+/**
+* @brief This test to get current video mode.
+*
+* This test function is to get current video mode of selected HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 007@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_video_mode(void)
+{
+    gTestID = 7;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    dsVideoPortResolution_t resolution;
+    
+    ret = dsHdmiInGetCurrentVideoMode(&resolution);
+    if(ret != dsERR_NONE)
+    {
+       UT_LOG_ERROR("Result: Failed dsHdmiInGetCurrentVideoMode Error:[%s]",ut_control_GetMapString(dsError_mapTable, ret));
+    }
+
+    UT_LOG_INFO("Result: Passed dsHdmiInGetCurrentVideoMode resolution name: %s, pixelResolution: %s\t"
+                      "aspectRatio:%s, stereoScopicMode:%s, frameRate:%s, interlaced:%s ",
+                      resolution.name,
+                      ut_control_GetMapString(dsVideoResolution_mapTable, resolution.pixelResolution),
```

Comment:
line no 835 - instead of values use enums 
   if(select < 0 || select >= dsHDMI_IN_PORT_MAX)
correct all places

Reply:
done